### ISSUE TITLE
Custom Skill not getting triggered

### ISSUE THREAD

**Issue by namang1029** [miguelmota/alexa-voice-service.js#12]
Everything is working fine if i ask any question but when i try to trigger my custom skill, it doesn't recognize. My custom skill is working fine on echosim.io. Also i tried from java swing application, i am able to call custom skill.

**Comment by mordka** [comment]
@namsak Have you resolved your issue?

**Comment by aSoujyuTanaka** [comment]
I have similar issue. When I invoke a custom skill by calling "ask (skill name)", alexa finds the skill and replies the correct answer. But following recording input for the custom skill is not recognized. It seems that the session is disconnected before each input. I tried alexa-avs-sample-app on Linux and can get the answer of the skill. I'd like to use this library for creating web UI.

**Comment by aSoujyuTanaka** [comment]
I found the skill session is made when I invoke the skill but it continues about 8 seconds only. https://stackoverflow.com/questions/38191837/how-to-keep-an-alexa-skill-open
For the time of echoing my utterance, it is almost impossible to tell alexa following command. I modified not to echo my voice but it is still hard to complete my question in that time. I suppose I could extend the time limit for another 8 seconds if reprompt message could be enabled, but I'm not sure this Library support that.
